### DISCUSSION TITLE
How to configure Test Runner for WebdriverIO with TypeScript on Webstrom IDE.

### DISCUSSION THREAD

**Discussion by aakashpaliwal13** [webdriverio/webdriverio#8745] [Community]
Hi, 

I am using WebdriverIO with TypeScript on Webstrom IDE.

I am unable to configure the testrunner on Webstrom.
<img width="1152" alt="Screenshot-2022-08-23-at-12 51 59-PM" src="https://user-images.githubusercontent.com/97433650/186113352-a065462e-ac10-43d4-891c-c4fb2c99d349.png">
<img width="1792" alt="Screenshot-2022-08-23-at-12 57 25-PM" src="https://user-images.githubusercontent.com/97433650/186113370-be3216b9-ca95-467b-a4a5-0699b1de5894.png">

**Comment by erwinheitzman** [comment]
Seems like you point to the cli package instead of the webdriverio package

**Comment by erwinheitzman** [comment]
This question isn't about Webstorm but about a Webstorm plugin (https://plugins.jetbrains.com/plugin/16147-webdriverio) which is maintained by a third party.
The code for this plugin can be found here https://github.com/winkingzhang/idea-run-wdio

If you need support for this you will have to contact the creator as we have got nothing to do with Webstorm plugins written by someone else.

**Reply by aakashpaliwal13** [reply]
Thanks @erwinheitzman
